data/strings/strings.txt

```diff
@@ -16958,6 +16958,98 @@
     zh-Hans = 关于 OpenStreetMap 的更多信息
     zh-Hant = 關於 OpenStreetMap 的更多資訊
 
+  [editor_osm_history]
+    tags = android
+    en = Your editing history
+    af = Jou redigeergeskiedenis
+    ar = سجل التحرير الخاص بك
+    az = Redaktə tarixçəniz
+    be = Ваша гісторыя рэдагавання
+    bg = Вашата история на редактиране
+    ca = El teu historial d'edició
+    cs = Vaše historie úprav
+    da = Din redigeringshistorik
+    de = Ihr Bearbeitungsverlauf
+    el = Το ιστορικό επεξεργασίας σας
+    es = Su historial de edición
+    et = Teie toimetamise ajalugu
+    eu = Zure edizio-historia
+    fa = سابقه ویرایش شما
+    fi = Muokkaushistoriasi
+    fr = Votre historique d'édition
+    he = היסטוריית העריכה שלך
+    hi = आपका संपादन इतिहास
+    hu = Az Ön szerkesztési előzményei
+    id = Riwayat pengeditan Anda
+    it = La sua storia di editing
+    ja = 編集履歴
+    ko = 편집 기록
+    lt = Jūsų redagavimo istorija
+    lv = Jūsu rediģēšanas vēsture
+    mr = तुमचा संपादन इतिहास
+    nb = Din redigeringshistorikk
+    nl = Uw bewerkingsgeschiedenis
+    pl = Państwa historia edycji
+    pt = O seu histórico de edições
+    pt-BR = Seu histórico de edição
+    ro = Istoricul dvs. de editare
+    ru = Ваша история редактирования
+    sk = Vaša história úprav
+    sv = Din redigeringshistorik
+    sw = Historia yako ya uhariri
+    th = ประวัติการแก้ไขของคุณ
+    tr = Düzenleme geçmişiniz
+    uk = Ваша історія редагувань
+    vi = Lịch sử chỉnh sửa của bạn
+    zh-Hans = 您的编辑历史
+    zh-Hant = 您的編輯歷史記錄
+
+  [editor_osm_notes]
+    tags = android
+    en = Your map data notes
+    af = Jou kaartdata notas
+    ar = ملاحظات بيانات الخريطة الخاصة بك
+    az = Xəritə məlumatı qeydləriniz
+    be = Заўвагі да дадзеных карты
+    bg = Бележки за данните на вашата карта
+    ca = Notes de les vostres dades del mapa
+    cs = Vaše poznámky k mapovým podkladům
+    da = Dine noter til kortdata
+    de = Ihre Hinweise zu den Kartendaten
+    el = Σημειώσεις δεδομένων του χάρτη σας
+    es = Sus notas de datos cartográficos
```

Comment:
Same, using first person:

"Mis notas de datos cartográficos"

May change the "datos cartográficos" for something simpler if anyone suggests it, but it can stay by now.

Reply:
Maybe "Mis notas del mapa"?